scripts/generate/clean-up.ts

```diff
@@ -2,7 +2,7 @@ import fs from 'fs';
 import path from 'path';
 import {exec} from 'child_process';
 
-export function cleanUp() {
+export function cleanUp(): void {
```

Comment:
👎 to any rule that requires explicit, unnecessary type annotations. IMO all they do is make the a simple extension more confusing for beginners, when we already give them all the benefits of TypeScript without needing almost any explicit types.

Reply:
I do agree, that devs unaccustomed to TS have an extra burdon.  So, as per the [earlier comment](https://github.com/Shopify/argo-checkout-template/pull/27#issuecomment-654472306), I'll modify it and disable the rule, departing from vanilla / recommended setup.

Reply:
Disabled `@typescript-eslint/explicit-module-boundary-types'